test/deps/closure_test.py

```diff
@@ -26,6 +30,30 @@ def test_add_item(self):
         self.assertEqual(ClosureItem(resource, version), c1['user/block/name1'])
         self.assertIn('Incompatible dependency', str(biiout))
 
+    def test_bii_config_deps(self):
```

Comment:
Not really this test. You want to make sure that when depending on a certain block, the biicode.conf is actually included in the Closure and you can access it via the Manager. Here you are actually adding biicode.conf manually to the closure, so pretty obvious that it will be included in the closure, so you are just testing a dict.